assets/js/sidebar/sidebar-list.js

```diff
@@ -8,15 +8,18 @@ const SIDEBAR_TYPE = {
   tasks: 'tasks'
 }
 
-const SIDEBAR_NAV_TYPES = [SIDEBAR_TYPE.extras, SIDEBAR_TYPE.modules, SIDEBAR_TYPE.tasks]
+const SIDEBAR_TAB_TYPES = [SIDEBAR_TYPE.extras, SIDEBAR_TYPE.modules, SIDEBAR_TYPE.tasks]
 
-const SIDEBAR_NODE_LIST_SELECTOR = '#full-list'
+const sidebarNodeListSelector = type => `#${type}-full-list`
 
 /**
  * Initializes the sidebar navigation list.
  */
 export function initialize () {
-  renderSidebarNodeList(getSidebarNodes(), getCurrentPageSidebarType())
+  SIDEBAR_TAB_TYPES.forEach(type => {
+    renderSidebarNodeList(getSidebarNodes(), type)
+  })
```

Comment:
Sidebar tabs were rewritten to have separate `ul` elements for each tab. I was following https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example as an example of a correctly implemented tab list, and each tab needs to have a different "tabpanel" component assigned to it. Hence multiple lists were necessary.